### ISSUE TITLE
Internal error after triggering a lengthy download of data

### ISSUE THREAD

**Issue by smathermather** [OpenDroneMap/odm360#143]
After triggering a lengthy download of data, I tried stopping the service and get:

> Internal Server Error
> The server encountered an internal error and was unable to complete your request. Either the server is overloaded or there is an error in the application.

My hunch is this is due to how we've launched flask and tied in with issue #132, but I thought I would open this here in case 132 isn't a fix, and my hunch is wrong

**Comment by smathermather** [comment]
Confirmed that I am able to toggle the service off after the download is complete, which strengthens my suspicion that this will be fixed by 132.

**Comment by hcwinsemius** [comment]
We can also disable download during the service. I don't think we should
download anyway during a survey.

Op di 17 nov. 2020 17:38 schreef Stephen Mather <notifications@github.com>:

**Comment by smathermather** [comment]
Yes, this probably should be changed. But I suspect a lot of functions fail during download, likely due to lack of concurrency support.
